src/data.js

```diff
@@ -1,24 +1,52 @@
-const data = [
-  { name: 'King007', score: 23 },
-  { name: 'PaulWalker', score: 18 },
-  { name: 'Gonakie', score: 16 },
-  { name: 'Punisher', score: 30 },
-  { name: 'Microscope', score: 24 },
-  { name: 'RealGuy', score: 13 },
+import { getScores, postScore } from './api-stuff.js';
+
+// eslint-disable-next-line import/no-mutable-exports
+let data = [
+  { user: 'King007', score: 23 },
+  { user: 'PaulWalker', score: 18 },
+  { user: 'Gonakie', score: 16 },
+  { user: 'Punisher', score: 30 },
+  { user: 'Microscope', score: 24 },
+  { user: 'RealGuy', score: 13 },
 ];
 
-const idFromIndex = (i) => `${data[i].name}-${data[i].score}`;
+const idFromIndex = (i) => `${data[i].user}-${data[i].score}`;
 
 function displayData(data) {
   const lbl = document.querySelector('.lb-list');
   lbl.innerHTML = '';
   const codeForData = data.sort((a, b) => b.score - a.score)
     .map((el, i) => `<li id="${idFromIndex(i)}" class="show">
                 <span class="rk">${i + 1}</span>
-                <span class="name">${el.name}</span>
+                <span class="user">${el.user}</span>
                 <span class="score">${el.score}</span>
               </li>`).join('');
   lbl.innerHTML = codeForData;
 }
```

Comment:
- [ ] Also you should refactor all the functions to the arrow functions.

Reply:
Done.